boxoffice/models/item_collection.py

```diff
@@ -48,15 +48,20 @@ def fetch_all_details(self):
         line_item_query = db.select([LineItem.id, LineItem.customer_order_id, Order.invoice_no, Item.title, LineItem.base_amount,
             LineItem.discounted_amount, LineItem.final_amount, DiscountPolicy.title, DiscountCoupon.code,
             Order.buyer_fullname, Order.buyer_email, Order.buyer_phone, Assignee.fullname,
-            Assignee.email, Assignee.phone, Assignee.details, OrderSession.utm_campaign,
+            Assignee.email, Assignee.phone, Order.access_token, Assignee.details, OrderSession.utm_campaign,
             OrderSession.utm_source, OrderSession.utm_medium, OrderSession.utm_term,
             OrderSession.utm_content, OrderSession.utm_id, OrderSession.gclid, OrderSession.referrer,
             Order.paid_at]).select_from(line_item_join).where(LineItem.status ==
             LINE_ITEM_STATUS.CONFIRMED).where(Order.item_collection ==
             self).order_by(LineItem.ordered_at)
 
         return HeadersAndDataTuple(
-            ['ticket_id', 'order_id', 'receipt_no', 'ticket_type', 'base_amount', 'discounted_amount', 'final_amount', 'discount_policy', 'discount_code', 'buyer_fullname', 'buyer_email', 'buyer_phone', 'attendee_fullname', 'attendee_email', 'attendee_phone', 'attendee_details', 'utm_campaign', 'utm_source', 'utm_medium', 'utm_term', 'utm_content', 'utm_id', 'gclid', 'referrer', 'date'],
+            ['ticket_id', 'order_id', 'receipt_no', 'ticket_type', 'base_amount',
+                'discounted_amount', 'final_amount', 'discount_policy', 'discount_code',
+                'buyer_fullname', 'buyer_email', 'buyer_phone', 'attendee_fullname',
+                'attendee_email', 'attendee_phone', 'assignee_url', 'attendee_details',
+                'utm_campaign', 'utm_source', 'utm_medium', 'utm_term', 'utm_content', 'utm_id',
+                'gclid', 'referrer', 'date'],
```

Comment:
This whole function is a bit messy. The mapping of column order and header name is manually done. Need to add a better way to manage this at some point.

Reply:
It should be possible to (a) assign labels in the select query and (b) extract those labels from the first row (or use `csv.DictWriter`, so the labels are extracted automatically).

Reply:
Try this: `db.select([LineItem.id.label('ticket_id'), LineItem.customer_order_id.label('order_id'), ...`